content/applications/hr/payroll/contracts.rst

```diff
@@ -65,41 +62,52 @@ Required fields
   - :guilabel:`PFI` is a Belgian-specific contract used when hiring employees that need training,
     and covers the training period specifically.
 
-- :guilabel:`End Date`: If the contract has a specific end date, click the drop-down menu, navigate
+- :guilabel:`End Date`: if the contract has a specific end date, click the drop-down menu, navigate
   to the correct month and year using the arrow icons, then click on the date.
-- :guilabel:`HR Responsible`: If there is a specific person in HR that is responsible for the
+- :guilabel:`HR Responsible`: if there is a specific person in HR that is responsible for the
   contract, select the person from the drop-down menu.
-- :guilabel:`Analytic Account`: This field allows a link between the contract and a specific
+- :guilabel:`Analytic Account`: this field allows a link between the contract and a specific
   analytic account for accounting purposes.
 
 Contract details
 ----------------
 
-The contract details section allows for the addition and editing of a contract, and the ability to
-send the contract to the employee for approval and signatures.
+The :guilabel:`Contract Details` tab allows for the addition and editing of a contract, along with
+specifying which template to use when a new contract is created. These fields **must** be populated
+in order to create a new contract.
+
+.. important::
+   To access the various contract template fields in the :guilabel:`Contract Details` tab, the
+   *Salary Configurator* (`hr_contract_salary`) module **must** be :ref:`installed
+   <general/install>`.
+
+   When the *Salary Configurator* module is installed, the *Salary Configurator - Holidays* and
+   *Salary Configurator - Payroll* modules install, as well.
+
+   Once the modules are installed, the database reverts to the main dashboard.
 
 .. image:: contracts/contract-details.png
    :align: center
    :alt: Contract details in optional tabs for a new contract.
 
-- :guilabel:`Contract Template`: Select a pre-existing contract template from the drop-down menu.
+- :guilabel:`Contract Template`: select a pre-existing contract template from the drop-down menu.
   Contract templates are typically created through the *Recruitment* application.
-- :guilabel:`New Contract Document Template`: Select a contract from the drop-down menu to be
+- :guilabel:`New Contract Document Template`: select a contract from the drop-down menu to be
   modified for this new employee contract.
-- :guilabel:`Contract Update Document Template`: Select a contract from the drop-down menu if the
+- :guilabel:`Contract Update Document Template`: select a contract from the drop-down menu if the
   employee has an existing contract that requires updating.
-- :guilabel:`Notes`: The notes field is a text field where any notes for the employee contract can
+- :guilabel:`Notes`: the notes field is a text field where any notes for the employee contract can
   be entered for future reference.
 
 Modifying a contract
```

Comment:
I *believe* there is an 'external link' icon available now -- might be nice to add that in this section below

Reply:
Also, can you be more specific where this external link button should be clicked? At this point, the reader is on a contract form, and if they're in 'edit' mode, they can click the external link on a few different fields... so, I think some elaboration is necessary here.

Reply:
Do you mean on the 'New Contract Document Template' field?